### PR TITLE
Fixes #6581

### LINKED ISSUE #6581: Ghosts Activating Washing Machines
(WEB REPORT BY: platinumlobster REMOTE: :7777)
Revision: f1ec4a92f99a9f5d4b39822a7a3ac5c9c8625db9

Ghosts can open and close washing machines, possibly popping out any items inside if they're full.

I saw something in a washing machine as an observer and was curious what it was, but figured I'd just have to wait until somebody came back and opened it to take what was inside.

I went and clicked on it as a ghost and I actually opened it and popped out the purple jumpsuit inside.

I can consistantly and always open and close washing machines that are visible anywhere on the screen as a ghost just by left clicking on them.

### PR DESCRIPTION
**Pull request by DeityLink**
Fixes #6581

### CODE REVIEW

**Comment by PJB3005** [comment]
If ghosts can actually use `attack_hand()`, the issue goes deeper than this bandaid.

**Comment by clusterfack** [comment]
Did you try using the ACTUAL standard for ghost hands? 
'''
If(..())
Return 1
'''

**Comment by clusterfack** [comment]
Darn, I tried

**Comment by PJB3005** [comment]
Nerd it's a \`

**Comment by clusterfack** [comment]
Both look the same on a phone

**Comment by DeityLink** [comment]
I found the problem. attack_ghost() on machines calls attack_ai(), which in turn calls attack_hand().
I've fixed it, now only admin ghosts can do that.

**Comment by clusterfack** [comment]
Jesus H. Christ deity just STOP.

Just use the goddamn code I talked about in attack hand

**Comment by DeityLink** [comment]
@clusterfack can you answer my questions on IRC pls?

File: code/modules/research/xenoarchaeology/artifact/artifact_replicator.dm [hunk 1]
```diff
@@ -118,7 +118,12 @@
 	else
 		src.visible_message("<span class='warning'>\The [user] taps \the [src] with \the [O].</span>")
 
+/obj/machinery/washing_machine/attack_ghost(mob/user as mob)
```
Comment (clusterfack) [inline code/modules/research/xenoarchaeology/artifact/artifact_replicator.dm]: What is this and why is it here

**Comment by DeityLink** [comment]
duh

**Comment by PJB3005** [comment]
100% sure the parent `attack_hand()` handles prints.

**Comment by clusterfack** [comment]
That's another reason why he should be calling if(..()) return, why aren't you deity?

**Comment by PJB3005** [comment]
aaaaaah:

``` DM
. = ..()
if(.)
    return
```

**Comment by DeityLink** [comment]
Except this alien artifact doesn't really need anything else in the parent proc. It doesn't check for NOPOWER/BROKEN, etc

**Comment by clusterfack** [comment]
Any child of machinery should have mine or pjb's version of sanity in attack hand unless they have a wildly exceptional reason to not have it

**Comment by DeityLink** [comment]
Done and squashed.

**Comment by clusterfack** [comment]
Close enough

**Pull request merged**